src/RRule.php

```diff
@@ -2054,15 +2064,20 @@ static protected function i18nLoad($locale)
 	/**
 	 * Format a rule in a human readable string
 	 * intl extension is required.
+	 *
+	 * @param array  $opt
+	 * @param string $fallbackLocale
+	 *
+	 * @return string
 	 */
-	public function humanReadable(array $opt = array())
+	public function humanReadable(array $opt = array(), $fallbackLocale = 'en_US')
```

Comment:
You don't need to add a new argument here, it should be part of the `$opt` array.